spec/system/candidate_interface/continuous_applications/course_selection/candidate_changes_course_and_reaches_limit_spec.rb

```diff
@@ -0,0 +1,84 @@
+require 'rails_helper'
+
+RSpec.feature 'Changing a course' do
+  include CandidateHelper
+
+  it 'Candidate changes course to one they have already been rejected from twice' do
+    given_i_am_signed_in
+    and_there_are_course_options
+    and_i_have_two_rejected_application_to_a_course
```

Comment:
`applications`

Reply:
https://github.com/DFE-Digital/apply-for-teacher-training/pull/9222/commits/ef2325b4469ba3f08f2e48b816c562f73d2b4e2e